docs/user/glossary.rst

```diff
@@ -39,18 +40,15 @@ toggle additional features using the :guilabel:`Tools` menu for each term.
 
 .. image:: /images/glossary-tools.png
 
-Not translatable terms
-++++++++++++++++++++++
+Untranslatable terms
+++++++++++++++++++++
 
 .. versionadded:: 4.5
 
-Glossary terms which are read-only are not meant to be translated. You can use
-this for names or other terms which should not change while translating. Such
-terms are visually highlighted in the glossary sidebar.
-
-The terms can be flagged using :guilabel:`Tools` ↓
-:guilabel:`Mark as read-only`. In the background this toggles the ``read-only``
-flag of the string.
+Flagging certain glossary term translations ``read-only`` manually, or
```

Comment:
No, manually is clicking the sidebar and typing in "read-only". Or by bulk-editing.
I didn't really go into it, because if you are editing manually, you should know.
I might be wrong there, but this keeps it easy on the ones that know nothing.

Reply:
hmmm... I definitely wouldn't call a "bulk-edit" a "manual" way... especially in comparison with pressing a button in the UI for a specific term...

For ones who know nothing it's a bad form: it's suggests that there is another way to do it and they will have to rush through all other docs to find, what it is and if they need it... yet, I can't see an easy wayto put in words without rephrasing the whole paragraph... probably to resemble more the previous form...

Reply:
Changed, however the ordering might be strange now. More natural to promote the click-menu option first?

Reply:
yeah, probably so... But, I still like the original variant more; IMHO it's much more straight-forward: It describes the semantics of the feature and only then suggests how to use it...

I'd also avoid direct mention of `bulk-edit` it just looks out-of-place and just rather add a reference to a flag-related chapter of the doc...

Reply:
closing as outdated